torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1145,17 +1145,18 @@ at::Tensor& AtenXlaType::expm1_(at::Tensor& self) {
   return self;
 }
 
-at::Tensor AtenXlaType::eye(int64_t n, const at::TensorOptions& options) {
+at::Tensor& AtenXlaType::eye_out(at::Tensor& out, int64_t n) {
   XLA_FN_COUNTER("xla::");
-  return eye(n, n, options);
+  XLATensor out_tensor = bridge::GetXlaTensor(out);
+  XLATensor::eye_out(out_tensor, n, n);
+  return out;
 }
 
-at::Tensor AtenXlaType::eye(int64_t n, int64_t m,
-                            const at::TensorOptions& options) {
+at::Tensor& AtenXlaType::eye_out(at::Tensor& out, int64_t n, int64_t m) {
```

Comment:
Do we need to do this?
It seems pretty confusing.
Sometimes overriding `foo()` is enough. Sometimes you need `foo()` and `foo_out()`. Sometimes only `foo_out()`.
Is there a random number generator to drive these decisions on the PT side? 😁